docs/cluster-deployment/croud.rst

```diff
@@ -0,0 +1,233 @@
+   ==> Success: Operation completed.
+   +--------------------------------------+------------------------+----------+--------------+--------------------------------------+-------------+--------------------------------------------------+
+   | id                                   | name                   | numNodes | crateVersion | projectId                            | username    | fqdn                                             |
+   |--------------------------------------+------------------------+----------+--------------+--------------------------------------+-------------+--------------------------------------------------|
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        1 | 4.8.1        | 952cd102-91c1-4837-962a-12ecb71a6ba8 | admin       | my-first-crate-cluster.eastus.azure.cratedb.net. |
+   +--------------------------------------+------------------------+----------+--------------+--------------------------------------+-------------+--------------------------------------------------+
+
+.. tip::
+    The minimum length of a password is 24 characters.
+
+.. note::
+    Complete documentation on clusters in Croud, including all the deployment
+    arguments, can be found `here`_.
+
+Scale a cluster
+===============
+
+After deploying, the next thing that you might want to do is scale your
+cluster. This allows you to either add more CPUs or more storage to your
+cluster.
+
+Scaling
+-------
+
+Required arguments:
+
+``--cluster-id``
+    The CrateDB cluster ID to use.
+
+``--unit``
+    The product scale unit to use.
+
+Example:
+
+.. code-block:: console
+
+   sh$ croud clusters scale \
+       --project-id 952cd102-91c1-4837-962a-12ecb71a6ba8 \
+       --cluster-id 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 \
+       --unit 1
+   +--------------------------------------+------------------------+----------+
+   | id                                   | name                   | numNodes |
+   |--------------------------------------+------------------------+----------|
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        2 |
+   +--------------------------------------+------------------------+----------+
+   ==> Info: Cluster scaling initiated. It may take a few minutes to complete the changes.
+   ==> Info: Status: SENT (Your scaling request was sent to the region.)
+   ==> Info: Status: IN_PROGRESS (Scaling up from 2 to 3 nodes. Waiting for new node(s) to be present.)
+   ==> Success: Operation completed.
+   +--------------------------------------+------------------------+----------+
+   | id                                   | name                   | numNodes |
+   |--------------------------------------+------------------------+----------|
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        3 |
+   +--------------------------------------+------------------------+----------+
+
+Storage expansion
```

Comment:
I would add an explanation here that expanding storage requires temporarily stopping the cluster.